### PR TITLE
Docs: QE review fixes

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/proc-send-receive.adoc [hunk 1]
```diff
@@ -29,7 +29,8 @@ Use the output as the host name in the following step.
 
 . Run the client:
 +
-[options="nowrap",subs=attributes+]
+[options="nowrap",subs="+quotes,attributes"]
 ----
-python client-example1.py -u amqps://user1:password@messaging.example1.com:443/myqueue
+python client-example1.py -u amqps://user1:password@_messaging.example1.com_:443/myqueue
```
Comment (lulf) [inline documentation/modules/proc-send-receive.adoc]: I think this might break the console help page which replaces the hostname with the actual hostname.

Comment (lulf) [reply documentation/modules/proc-send-receive.adoc]: @jenmalloy i will build and check the console, so hold off merging until i've verified.

Comment (lulf) [reply documentation/modules/proc-send-receive.adoc]: @jenmalloy Help page looks good. Ship it!

Comment (jenmalloy) [reply documentation/modules/proc-send-receive.adoc]: @lulf Thank you!

**Pull request merged**